recipe/meta.yaml

```diff
@@ -34,9 +40,6 @@ test:
     - retriever.engines
     - retriever.lib
 
-  commands:
-    - retriever --help
```

Comment:
Looks like this failed because `argcomplete` was not present in `requirements/run`.

Reply:
Does this still fail now that `argcomplete` is a run time requirement?

Reply:
Waiting for the tests to complete, but it looks like the test are building past that point